src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -531,6 +515,10 @@ private TestLspServer(TestWorkspace testWorkspace, Dictionary<string, IList<LSP.
                     ExceptionStrategy = ExceptionProcessing.ISerializable,
                 };
 
+                // Workspace listener events do not run in tests, so we manually register the lsp misc workspace.
```

Comment:
Is that intentional? Is that something we can change?

Reply:
IIRC it is intentional, I think to prevent other things (like sln crawler) from automatically spinning up in tests. cc @tmat